src/app/public/plugin-resources/code-examples/checkbox/checkbox-demo.component.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-checkbox-demo',
+  templateUrl: './checkbox-demo.component.html'
+})
+export class CheckboxDemoComponent {
+
+  public checkboxItems: any[] = [
+    {
+      label: 'Checkbox 1'
```

Comment:
Should we update this code sample to match the demo?

Reply:
Done.